about.html

```diff
@@ -1,73 +1,109 @@
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
-        </nav>
-      </footer>
-    </div><!-- container -->        
-</body>
-</html>
\ No newline at end of file
+	<head>
+		<meta charset="utf-8" />
+
+		<title>Sprint Challenge - About</title>
+
+		<link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet" />
+		<link rel="stylesheet" href="css/index.css" />
+	</head>
+
+	<body>
+		<div class="container about-page">
+			<header>
+				<nav>
+					<img src="img/lambda-black.png" alt="Lambda Logo" />
+					<!-- <div class="links"> -->
+					<a href="index.html">Home</a>
+					<a href="about.html">About</a>
+					<a href="#">Products</a>
+					<a href="#">Blog</a>
+					<a href="#">Contact</a>
+					<!-- </div> -->
+				</nav>
+				<img class="header_img" src="img/jumbo-about.png" alt="" />
+			</header>
+
+			<div class="text-container added_border">
+				<h2>About</h2>
+				<p>
+					Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative
+					approaches to corporate strategy foster collaborative thinking to further the overall value
+					proposition. Organically grow the holistic world view of disruptive innovation via workplace
+					diversity and empowerment.
+				</p>
+			</div>
+
```

Comment:
The about page is a great place to use semantic HTML to describe the content area of the page.